apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -4,29 +4,39 @@ import {
 	useAppSelector,
 	useEffect,
 } from "~/libs/hooks/hooks.js";
+import { actions as groupActions } from "~/modules/groups/groups.js";
 import { actions as userActions } from "~/modules/users/users.js";
 
+import { getGroupColumns } from "./libs/helpers/get-group-columns.helper.js";
+import { getGroupRows } from "./libs/helpers/get-group-rows.helper.js";
 import { getUserColumns } from "./libs/helpers/get-user-columns.helper.js";
 import { getUserRows } from "./libs/helpers/helpers.js";
```

Comment:
Import all helpers into a file helpers.ts and import everything from one place
```suggestion
import { getUserRows, getUserColumns, getGroupColumns, getGroupRows } from "./libs/helpers/helpers.js";
```